electron/constants.ts

```diff
@@ -109,6 +114,18 @@ function getSteamCompatFolder() {
   return `${userHome}/.steam/steam`
 }
 
+export function getSteamLibraries() {
+  const vdfFile = join(steamCompatFolder, 'steamapps', 'libraryfolders.vdf')
+  if (existsSync(vdfFile)) {
+    const json = parse(readFileSync(vdfFile, 'utf-8'))
+    const folders = Object.values(json.libraryfolders) as Array<{
+      path: string
+    }>
+    return folders.map((folder) => folder.path)
+  }
+  logDebug('No other steam libraries found', LogPrefix.Backend)
```

Comment:
This log could be clearer, something like `Unable to load Steam Libraries, libraryfolders.vdf not found`

Reply:
done